versioned_docs/version-3.0.0-beta/personal-accounts/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+slug: /personal-accounts/overview
```

Comment:
please update the filename to lowercase - `overview.md`

Reply:
Addressed